File: hindsight-tools/hindsight-fs/src/paths.ts
/** Shared filesystem layout constants for a mount directory. */

/** Hidden control sub-directory inside every mount (config, state, daemon). */
export const CONTROL_DIR = ".hindsight-fs";

export const STATE_FILE = "state.json";
export const PID_FILE = "daemon.pid";
export const LOG_FILE = "daemon.log";
export const INDEX_FILE = "index.md";

File: hindsight-cli/src/commands/fs/paths.rs
//! Shared filesystem layout constants for a mount directory.

/// Hidden control sub-directory inside every mount (config, state, daemon).
pub const CONTROL_DIR: &str = ".hindsight-fs";

pub const STATE_FILE: &str = "state.json";
pub const PID_FILE: &str = "daemon.pid";
pub const LOG_FILE: &str = "daemon.log";
pub const INDEX_FILE: &str = "index.md";
